v1/openapi/schemas/code-validation-error.yaml

```diff
@@ -4,13 +4,15 @@ type: object
 properties:
   code:
     type: string
+    description: Error code received from the backend services.
   message:
     type: string
+    description: Short error description.
   description:
     type: string
-  link:
-    type: string
+    description: Additional information about the error.
   validation_failures:
+    description: Array containing incorrect parameters/fields.
```

Comment:
Suggestion: "...parameters/properties". Since the actual object field is named "property".

Reply:
thx